src/classes/ALLO_Allocations_TEST.cls

```diff
@@ -531,7 +508,7 @@ private with sharing class ALLO_Allocations_TEST {
         setupSettings(new Allocations_Settings__c(Default_Allocations_Enabled__c = true, Default__c = defaultGau.id));
 
         //setting flags to not create default allocations for the new opportunities
-        ALLO_Allocations_TDTM.resetFlags(true);
+        TDTM_TriggerHandler.toggleTriggerState('Opportunity', 'ALLO_Allocations_TDTM', false);
```

Comment:
I'm torn on this.  i agree it is more explicit, but I suspect that some of the other recursive flag uses could also be accomplished this way.  I'm also concerned that people might start following this pattern and doing a set and forget, which seems dangerous given our static flag issues.  I realize this is just test code, but you do later use this method in production code.  @njjc thoughts?